desktop/cmp/filter/PopoverFilterChooser.ts

```diff
@@ -0,0 +1,104 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2025 Extremely Heavy Industries Inc.
+ */
+
+import {hoistCmp, HoistModel, lookup, useLocalModel, uses} from '@xh/hoist/core';
+import {bindable, makeObservable} from '@xh/hoist/mobx';
+import {box, hframe} from '@xh/hoist/cmp/layout';
+import '@xh/hoist/desktop/register';
+import {popover} from '@xh/hoist/kit/blueprint';
+import {getLayoutProps} from '@xh/hoist/utils/react';
+import './PopoverFilterChooser.scss';
+import {filterChooser, FilterChooserProps} from './FilterChooser';
+import {FilterChooserModel} from '@xh/hoist/cmp/filter';
+
+/**
+ * A wrapper around a FilterChooser that renders in a popover when opened, allowing it to expand
+ * vertically beyond the height of a toolbar.
+ * @see FilterChooser
+ */
+export const [PopoverFilterChooser, popoverFilterChooser] =
+    hoistCmp.withFactory<FilterChooserProps>({
+        model: uses(FilterChooserModel),
+        className: 'xh-popover-filter-chooser',
+        render({model, className, ...props}, ref) {
+            const layoutProps = getLayoutProps(props),
+                impl = useLocalModel(PopoverFilterChooserLocalModel);
+
+            return box({
+                ref,
+                className,
+                ...layoutProps,
+                item: popover({
+                    isOpen: impl.popoverIsOpen,
+                    popoverClassName: 'xh-popover-filter-chooser__popover',
+                    item: hframe(
+                        filterChooser({
+                            model,
+                            // Omit when popover is open to force update the inputRef
+                            omit: impl.popoverIsOpen,
+                            disabled: true,
+                            className: 'xh-popover-filter-chooser__filter-chooser',
+                            displayCount: true,
+                            ...props
```

Comment:
Should we move `disabled: true` below the spread, in case a dev were to explicitly pass `disabled: false` as a prop?